src/world/generator/PopulationTask.php

```
@@ -29,130 +9,91 @@
+            [$dx, $dz] = World::getXZ($hash);
+            $adjChunks[$hash] = self::setOrGenerate($mgr, $gen, $this->chunkX + $dx, $this->chunkZ + $dz, $chunkObj);
+        }
+
+        $gen->populateChunk($mgr, $this->chunkX, $this->chunkZ);
+
+        $popChunk = $mgr->getChunk($this->chunkX, $this->chunkZ)
+            ?? throw new AssumptionFailedError("Generated chunk missing");
+        $popChunk->setPopulated();
+
+        $this->centerChunk = FastChunkSerializer::serializeTerrain($popChunk);
+        $this->adjacentChunksData = igbinary_serialize(array_map(
```

PHPStan (2.1.15)

Property pocketmine\world\generator\PopulationTask::$adjacentChunksData (string) does not accept string|null.


(assign.propertyType)